setup_env.bat

```diff
@@ -0,0 +1,8 @@
+@echo off
+call ./scripts/win/install-protobuf.bat
```

Comment:
getting this error

setup_env.bat
Access is denied.
Exception calling "ExtractToDirectory" with "2" argument(s): "Could not find file
'C:\Users\User\AppData\Local\Temp\protoc-3.6.1-win32.zip'."
At line:1 char:53
+ ... ileSystem'; [IO.Compression.ZipFile]::ExtractToDirectory('C:\Users\Us ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : FileNotFoundException


SUCCESS: Specified value was saved.
go: finding github.com/golang/protobuf/protoc-gen-go latest

Reply:
Odd I can't reproduce and didn't require administrator; but will change the download to use something other than certutil - it's a bit of an odd hack I suspect it is the cause.